src/components/calcite-pick-list-group/resources.ts

```diff
@@ -1,4 +1,5 @@
 export const CSS = {
   heading: "heading",
-  container: "container"
+  container: "container",
+  indent: "indent"
```

Comment:
Nitpick: If this is only a modifier for the container, would `indentedContainer` (`container--indented`) or similar better convey its usage?

Reply:
@jcfranco I address this in #783.